LanguageFeatures/Augmentation-libraries/augmenting_constructors_A19_t02.dart

```diff
@@ -32,6 +32,12 @@ class D extends C {
   D(super.x, [super.y = 0]);
 }
 
+extension type ET(int x) {
+  ET.foo(this.x);
+  factory ET.bar(int x) = ET;
+  factory ET.baz(int x) = ET.foo;
```

Comment:
I'm pushing for an error in dart-lang/language#4157. It seems inconsistent that it's an error to replace `super.name1(42)` by `super.name2('Hello')` in an augmenting declaration, but redirections can be replaced freely.

I think the wording in the feature spec has strong hints that this error was intended all the time.

Reply:
4157 has landed.